.github/workflows/data-product-prison-example-test.yaml

```diff
@@ -30,8 +30,10 @@ jobs:
 
       - name: Install Python dependencies
         run: |
-          pip install --requirement requirements.txt
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
 
-      - name: Run tests with pytest
-        run: |
-          pytest tests/ -vv;
+      - name: Install plugin
+        run: pip install pytest-github-actions-annotate-failures
```

Comment:
Can this be put into another requirements file with a version? I'm going to be rolling out Dependabot for Pip soon, so would be good to get this under its robotic remit 🤖